converters/video_to_video/video_to_video.py

```diff
@@ -2,30 +2,17 @@
 import os
 from converters.converter import Converter
 
-class VideoConverter(Converter):
+
+class VideoToVideoConverter(Converter):
     def __init__(self, video_path):
         super().__init__(video_path)
 
-    def to_frames(self, output_path=None, fps=1):
-        """Convierte el video en una serie de fotogramas."""
-        frames_folder = os.path.join('outputs', 'video_to_frames_outputs', self.filename)
-        os.makedirs(frames_folder, exist_ok=True)
-
-        if output_path is None:
-            output_path = os.path.join(frames_folder, '%04d.jpg')
-
-        # Comando ffmpeg para extraer los fotogramas del video
-        ffmpeg.input(self.file_path).filter('fps', fps=fps).output(output_path).run(overwrite_output=True)
-        return frames_folder
-
-    def to_format(self, output_format=None, fps=None, video_codec=None, audio_codec=None, audio_channels=None):
-        """Convierte el video al formato especificado con opciones."""
+    def convert(self, output_format=None, fps=None, video_codec=None, audio_codec=None, audio_channels=None):
```

Comment:
I think the parameters of the method should follow the rule set in the abstract method using **kwargs as is being done in audio convert